src/LoggerInterface.php

```diff
@@ -22,104 +22,104 @@ interface LoggerInterface
     /**
      * System is unusable.
      *
-     * @param string  $message
+     * @param string|\Stringable $message
```

Comment:
shouldn't those phpdoc changes be applied in the 1.x version too if the spec asks to support them ?

Reply:
That would break static analysis under PHP < 8 though... Unless you officially require the Symfony polyfill.

Reply:
not adding it would make static analysis complain when passing an object with `__toString`

Reply:
What I do in these cases is: `require-dev` Symfony polyfill, and add it to `suggest` too - this way the tools are happy, and the consumers get a heads-up that if they want to use stringables, they need the polyfill too. Avoids that dependency on the polyfill in my packages.